jsonnet/versions.yaml

```diff
@@ -22,3 +22,4 @@ versions:
   prometheusAdapter: 0.10.0
   prometheusOperator: 0.63.0
   thanos: 0.30.2
+  consolePlugin: 4-13
```

Comment:
```suggestion
  consolePlugin: 4.13
```
Or is `4-13` intentional for testing with Andy's image build?

Reply:
yes, that is intentional (for now) to use Andy's image.

Reply:
@jan--f just noting that before this patch goes in, we will have to change this to match the image tag that @kyoto is publishing.